tools/docgen/src/custom_markdown_documenter.ts

```diff
@@ -876,14 +876,21 @@ export class MarkdownDocumenter {
               ])
             );
           } else {
-            propertiesTable.addRow(
-              new DocTableRow({configuration}, [
-                this._createTitleCell(apiMember, true),
-                this._createModifiersCell(apiMember),
-                this._createPropertyTypeCell(apiMember),
-                this._createDescriptionCell(apiMember),
-              ])
-            );
+            if (
+              !(
+                ApiProtectedMixin.isBaseClassOf(apiMember) &&
```

Comment:
is this needed? can you mark the protected member as `@internal` instead?

Reply:
Those are internal by definition. We should not expose them in the docs. 
But happy to reconsider if there is an use case.

Reply:
This is not an universal rule I think. In general, we could expose some protected members so that the users can subclass from it (e.g., like in transport classes).